src/main/java/com/example/controller/TicketController.java

```diff
@@ -0,0 +1,25 @@
+package com.example.controller;
+
+import com.example.models.Ticket;
+import com.example.services.TicketService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/ticket")
+public class TicketController {
+
+    @Autowired
+    TicketService ticketService;
```

Comment:
1. inject beans with the help of constructor injection, not with field injection
2. spring beans have to be `private final`